components/builder-originsrv/src/server/mod.rs

```diff
@@ -152,7 +152,6 @@ impl Dispatcher for OriginSrv {
         router_pipe: Arc<String>,
     ) -> SrvResult<<Self::State as AppState>::InitState> {
         let state = ServerState::new(config, router_pipe)?;
-        state.datastore.setup()?;
         state.datastore.start_async();
```

Comment:
We can't just remove the datastore setup completely.. there is some async registration that happens at the end of setup in originsrv that is still needed.

Reply:
Ah, you're right.  Good catch.